api/user.go

```diff
@@ -207,7 +207,7 @@ func fireAndForgetWelcomeEmail(name, email, teamName, link string) {
 
 		subjectPage := NewServerTemplatePage("welcome_subject", link)
 		bodyPage := NewServerTemplatePage("welcome_body", link)
-		bodyPage.Props["FullName"] = name
+		bodyPage.Props["Nickname"] = name
```

Comment:
Nothing in the template files reference FullName or any of variation of it so I don't believe those properties are currently used anyway.